src/pluto/pluto.py

```diff
@@ -395,17 +396,56 @@ def create_null(df_null, v1, v2, v3, condo, mapped):
                 "pfirm15_flag",
             ]
 
-            def generate_graph(r, total, title):
-                y = [r[i] for i in x]
-                text = [f"{round(r[i]/total*100, 2)}%" for i in x]
-                return go.Scatter(
-                    x=x,
-                    y=y,
-                    mode="lines",
-                    name=title,
-                    hovertemplate="<b>%{x} %{text}</b>",
-                    text=text,
+            field_range = st.slider(
+                "Select a range of values",
+                min_value=0,
+                max_value=len(x),
+                value=[0, 12],
+            )
+
+            def sort_and_filter_df(df, range):
+                df = df[["pair"] + df.columns.tolist()[range[0] + 1 : range[1] + 1]]
+                df_transformed = (
```

Comment:
Can we break this up into multiple lines? I know the chained pandas methods are readable for good programmers but we aren't that smart 😞